benchmark/_http-benchmarkers.js

```diff
@@ -112,7 +112,7 @@ exports.run = function(options, callback) {
   child.stderr.pipe(process.stderr);
 
   let stdout = '';
-  child.stdout.on('data', (chunk) => stdout += chunk.toString());
+  child.stdout.on('data', (chunk) => { return stdout += chunk.toString(); });
```

Comment:
I actually object to this code fairly strongly. This has gone beyond style, to writing code that no longer does what it appears to do:

> Just to clarify, when you use braces, an explicit return is required when returning a value no matter what. If you call a function that is defined like () => { 'foo' }, it will return undefined.

The above is basic javascript: functions return undefined unless they return an explicit value, rocket functions or function functions are identical with respect to this.

Event Emitter listener return values are discarded, forcing people to return values that will be discarded is actually confusing, it makes it look the return value is used, when it is not.